### ISSUE TITLE
Specify export location

### ISSUE THREAD

**Issue by dt192** [GoomiiV2/Rampancy#24]
It would be good if we could specify an export location at the level creation stage rather than being forced to use solo or multi.
I use `data\levels\custom\mapname` for my own stuff and someone I'm helping has theirs grouped under their own parent folder too.

**Comment by GoomiiV2** [comment]
Fair point, the create level popup need abit of a polish pass so I'll add this to it then as well.

Thinking to have the custom folder be a drop down of existing folders that are already present under ``data\levels``, idea being to help avoid tedious retyping or typos.

**Comment by GoomiiV2** [comment]
Ok this is added in the Halo3 branch, I hope to fix up the Halo 1 support and merge to main for a release soon.
![image](https://user-images.githubusercontent.com/5684215/189514559-91e356c7-6f75-41eb-aa6a-f69c6b16066f.png)

**Comment by dt192** [comment]
That seems better, I was going to suggest moving the game selection to the level creation too.

Other thing I noticed is that the instruction on the page say to download `Unity 2021.2.0b4`, but when I try to load the project it asks me for `2022.1.0b16`, also the instructions say `Open the project (Rampancy/Unity directory) in Unity` but the directory is called `Rampancy/RampancyUnity`

**Comment by GoomiiV2** [comment]
Ah, been doing some refactoring and streamline in the dev branch (https://github.com/GoomiiV2/Rampancy/tree/Halo3) so the main ones abit behind and wrong now.

I need to fix up a few more things before I can merge that Halo 3 Branch to main.

**Comment by Redir1** [comment]
> Ah, been doing some refactoring and streamline in the dev branch (https://github.com/GoomiiV2/Rampancy/tree/Halo3) so the main ones abit behind and wrong now.
> 
> I need to fix up a few more things before I can merge that Halo 3 Branch to main.

I hope this wip will get it's f, hope you come back.
